### ISSUE TITLE
DAC 7 - High Priority WCAG Level A - Sensory Characteristics 02

### ISSUE THREAD

**Issue by JadeneAd** [ONSvisual/income-scrolly#7]
<img width="422" alt="DAC_Sensory_Characteristics_02" src="https://user-images.githubusercontent.com/75850106/155550650-74c74f2d-8622-41b2-9683-f1f4002b437e.png">

### Issue
Users are instructed to hover over highlighted areas. These areas can solely be found if the user has vision and is not colour deficient. This means that screen reader users and users that have difficulty perceiving colour may not be able to find the highlighted content. There is more than one instance in the page where highlighting the map is relied on.

<img width="596" alt="DAC 7 Current code ref" src="https://user-images.githubusercontent.com/75850106/155550898-c2552f6d-4030-4d6a-a01b-6e59d6fca410.png">

**Screen reader comments:**
“I examined the content in context under the heading ‘More income-deprived’, and was presented with the instruction ‘Hover over the highlighted areas to see their income deprivation distributions’.
This was not an instruction I could follow, as I did not know what areas of the page were being referred to, or what form the highlighting took. Moreover, hovering was not open to me as I use a keyboard rather than a mouse to interact with the page, so I had no idea how to accomplish the same result.”

### Suggested Solution
There should be a non-text alternative for the highlighted areas instruction. We recommend that a table is provided that provides which neighbourhoods are more towards the deprived end of the scale.
